packages/cli/src/commands/codegen.ts

```diff
@@ -139,12 +144,37 @@ async function run(options: Required<CodegenCommandOptions>) {
         project,
       });
 
-  const result = await codeGenerator.generate();
+  const execute = async (): Promise<boolean> => {
+    let result = false;
+
+    try {
```

Comment:
There is already a try-catch block within `codeGenerator.generate()` that catches and logs any errors, so the try-catch block in `codegen.ts` will log the errors a second time. 

https://github.com/polywrap/toolchain/blob/origin-dev/packages/cli/src/lib/codegen/CodeGenerator.ts

Reply:
Hmmm, this try/catch logic is also in the build command. Honestly I think it's alright to log more than once, and it might be best to remove the try/catch from within the `lib` layer. Thoughts?

Reply:
Yeah, either way is good. I prefer removing the try-catch from the lib layer if there are no side effects from doing that, but I'd also be happy merging this as-is.